mailpoet/lib/Util/DataInconsistency/DataInconsistencyRepository.php

```diff
@@ -24,6 +24,17 @@ public function getOrphanedSendingTasksCount(): int {
     return (int)$this->buildOrphanedSendingTasksQuery($builder)->getSingleScalarResult();
   }
 
+  public function getOrphanedScheduledTasksSubscribersCount(): int {
+    $stTable = $this->entityManager->getClassMetadata(ScheduledTaskEntity::class)->getTableName();
+    $stsTable = $this->entityManager->getClassMetadata(ScheduledTaskSubscriberEntity::class)->getTableName();
+    $count = $this->entityManager->getConnection()->executeQuery("
+      SELECT count(*) FROM $stsTable sts
+      LEFT JOIN $stTable st ON st.`id` = sts.`task_id`
+      WHERE st.`id` IS NULL
```

Comment:
I noticed some methods getting orphaned rows containing a condition with `OR`. Maybe we can add here a check that the subscriber exists. What do you think?

Reply:
I thought about it, but I see that when deleting a subscriber, we keep (I assume intentionally) the rows in task subscribers so that we display correct logs about sent emails.